src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectionTestWithSSLCert/CertificateTest.cs

```diff
@@ -13,6 +13,7 @@
 using System.Security.Cryptography.X509Certificates;
 using System.ServiceProcess;
 using System.Text;
+using Microsoft.SqlServer.TDS.Login7;
```

Comment:
This using is not used.
```suggestion
```

Reply:
I used "Remove and Sort Using" utility to get rid of unused reference and sort all usings.